drafts/developer-guide/pages/data-access-layer/module-config.adoc

```diff
@@ -0,0 +1,38 @@
+include::ROOT:partial$variables.adoc[]
+
+= Compose Module configuration.
+
+To configure your database connection at module level, navigate to your {PRODUCT_NAME} instance ( http://qc.cortezaproject.org/) and click the *Low Code application*.
+
+.The screenshot indicates how to access the low code application.
+image::access-control/adminn-area.png[role="data-zoomable"]
+
+A new screen showing you all of the available namespaces will appear. Select your preferred namespace.
+
+. From there depending on the namespace chosen
+
+. Navigate to the *[Admin panel]* and click on it.
+
+. Click on *[Module]* dropdown to choose an existing module or click *[New Module]* to create new module.
+
+. Fill in the field mappings forms and save.
```

Comment:
Can you clarify what this is referring to? Is it the module fields on this screenshot?

![image](https://user-images.githubusercontent.com/12545711/192238166-4372f581-5876-4559-b602-24f87bd0d77f.png)

Reply:
Please explain